### PR TITLE
npm run nuke + plylog 1.0.0 update

### PR DESCRIPTION
**Pull request by usergenic**
Needed to update stuff, including plylog.  Here is that story in PR form.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,10 +15,10 @@
     "test": "npm run build && npm run test:unit",
     "test:all": "npm run test:unit && npm run test:integration",
     "test:all:parallel": "npm run test:unit && npm run test:integration:parallel",
-    "test:integration": "lerna run test:integration --stream",
+    "test:integration": "lerna run test:integration --stream --concurrency 1",
     "test:integration:parallel": "lerna run test:integration --stream --parallel",
     "test:integration:windows": "lerna run test:integration --stream --ignore web-component-tester --ignore @polymer/esm-amd-loader --parallel",
-    "test:unit": "lerna run test:unit --stream",
+    "test:unit": "lerna run test:unit --stream --concurrency 1",
```
Comment (rictic) [inline package.json]: how does this affect our testing performance?

File: packages/esm-amd-loader/test/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   },
   "devDependencies": {
     "@polymer/esm-amd-loader": "^1.0.0",
-    "@webcomponents/webcomponentsjs": "^1.2.0",
+    "@webcomponents/webcomponentsjs": "^1.2.4",
```
Comment (rictic) [inline packages/esm-amd-loader/test/package.json]: Did you file a bug against webcomponentsjs?

Comment (rictic) [reply packages/esm-amd-loader/test/package.json]: Ah, looks like an IE bug was fixed in 1.2.6

File: package.json (hunk 1 above)
Comment (usergenic) [reply package.json]: Unit test time impact is negligible and it simplifies output.  If higher concurrency we got a real bump in test time on travis without impacting reliability I'd bump it up.

Comment (usergenic) [reply package.json]: (by negligible I mean not consistently more-than it was.)

**Pull request merged**